autode/wrappers/G09.py

```diff
@@ -652,7 +652,11 @@ def hessian_from(
 
         for line in reversed(calc.output.file_lines):
 
-            if r"\\@" in line or line.startswith(" @"):
+            if (
+                r"\\@" in line
+                or line.startswith(" @")
+                or line.startswith(" \@")
```

Comment:
I think it might be good to make this  r" \@" to be on the safe side, although on my current python version, it does not show any issues.

Reply:
💯 agree. thanks